### ISSUE TITLE
Rendering an array of hashes

### ISSUE THREAD

**Issue by anthonytokosky** [nesquena/rabl#474]
I am trying to render an array of hashes. My array looks like this:

days = [{:date=>Sun, 26 May 2013, :this_month=>false, :events=>{}}, {:date=>Mon, 27 May 2013, :this_month=>false, :events=>{}}]

I am trying to get an output that looks something like this:

[
  {
    date: "2013-05-26",
    this_month: false,
    events: { }
  },
  {
    date: "2013-05-27",
    this_month: false,
    events: { }
  }
]

but instead I'm getting this:

[
  {
    2013-05-26: {
      date: "2013-05-26",
      this_month: false,
      events: { }
    }
  },
  {
    2013-05-26: {
      date: "2013-05-27",
      this_month: false,
      events: { }
    }
  }
]

Basically I am trying to get rid of that first date that is repeated for every hash instance. 

My view currently:

collection @days

node do |day|
  day
end

And when I try a different method as described here:

https://github.com/nesquena/rabl/issues/141

object false

node :days do |day| 
  @days.map do |day|
    { :date => day.date, :this_month => day.this_month, :events => day.events}
  end
end

I get this error: 

undefined method `date' for {:date=>Sun, 26 May 2013, :this_month=>false, :events=>{}}:Hash

Any help would be very greatly appreciated.
